### PR TITLE
Refactor into three different projects

### PR DESCRIPTION
**Pull request by aggFTW**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,11 @@ python:
     - "2.7"
     - "3.3"
 install:
-    - pip install -r requirements.txt -e .
+    - pip install -r hdijupyterutils/requirements.txt -e hdijupyterutils/.
```
Comment (msftristew) [inline .travis.yml]: The trailing `/.` makes no sense

File: hdijupyterutils/setup.py [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-AUTHOR              = "Jupyter Development Team",
-AUTHOR_EMAIL        = "jupyter@googlegroups.org",
-URL                 = 'https://github.com/jupyter-incubator/sparkmagic'
-DOWNLOAD_URL        = 'https://github.com/jupyter-incubator/sparkmagic'
+DESCRIPTION         = "HdiJupyterUtils: Utils for Jupyter projects from HDInsight team"
+NAME                = "hdijupyterutils"
+PACKAGES            = ['hdijupyterutils']
+AUTHOR              = "Jupyter Development Team"
+AUTHOR_EMAIL        = "jupyter@googlegroups.org"
+URL                 = 'https://github.com/jupyter-incubator/hdijupyterutils'
```
Comment (msftristew) [inline hdijupyterutils/setup.py]: Change this back

File: autovizwidget/setup.py [hunk 3]
```diff
@@ -0,0 +1,63 @@
+DESCRIPTION         = "AutoVizWidget: An Auto-Visualization library for pandas dataframes"
+NAME                = "autovizwidget"
+PACKAGES            = ['autovizwidget', 'autovizwidget/plotlygraphs', 'autovizwidget/widget']
+AUTHOR              = "Jupyter Development Team"
+AUTHOR_EMAIL        = "jupyter@googlegroups.org"
+URL                 = 'https://github.com/jupyter-incubator/autovizwidget'
```
Comment (msftristew) [inline autovizwidget/setup.py]: Change this back

File: .travis.yml (hunk 1 above)
Comment (aggFTW) [inline .travis.yml]: True! Sorry, was just following a pattern.

**Comment by msftristew** [comment]
LGTM once you make those changes

File: hdijupyterutils/setup.py (hunk 2 above)
Comment (aggFTW) [inline hdijupyterutils/setup.py]: Good catch!

File: autovizwidget/setup.py (hunk 3 above)
Comment (aggFTW) [inline autovizwidget/setup.py]: Done.

**Pull request merged**